libraries/Microsoft.Bot.Builder.Azure.Queues/Schemas/AzureQueues.ContinueConversationLater.schema

```diff
@@ -30,23 +30,6 @@
                 "=addHours(utcNow(), 1)"
             ]
         },
-        "connectionString": {
-            "$ref": "schema:#/definitions/stringExpression",
-            "title": "Connection String",
-            "description": "Connection string for connecting to an Azure Storage account.",
-            "examples": [
-                "=settings.connectionString"
-            ]
-        },
-        "queueName": {
-            "$ref": "schema:#/definitions/stringExpression",
-            "title": "Queue Name",
-            "description": "Name of the queue that your azure function is bound to.",
-            "examples": [
-                "activities"
-            ],
-            "default": "activities"
```

Comment:
It looks like the class was changed to match the schema name is good, but if we have shipped this it is a breaking change.  Have we shipped it?

Reply:
It shipped as a preview, so we should be safe to change it.

---
In reply to: [501232641](https://github.com/microsoft/botbuilder-dotnet/pull/4692#discussion_r501232641) [](ancestors = 501232641)